src/pages/index.js

```diff
@@ -18,28 +20,77 @@ const profileTitleInput = document.querySelector("#profile-title-input");
 const profileDescriptionInput = document.querySelector(
   "#profile-description-input"
 );
+const profileImageContainer = document.querySelector(
+  ".profile__image-container"
+);
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "fd75a5c6-0c34-4591-90ec-6f1fc1e0d0b3",
+    "Content-Type": "application/json",
+  },
+});
 
+profileImageContainer.addEventListener("click", () => {
+  editAvatarPopup.open();
+});
 const addCardFormValidator = new FormValidator(configItems, addCardFormElement);
 const profileEditValidator = new FormValidator(configItems, profileEditForm);
+const editAvatarFormValidator = new FormValidator(
+  configItems,
+  document.querySelector("#avatar-edit-form")
+);
 
 addCardFormValidator.enableValidation();
 profileEditValidator.enableValidation();
+editAvatarFormValidator.enableValidation();
 
 const cardList = new Section(
   {
-    items: initialCards,
+    items: [],
     renderer: (item) => {
       const cardElement = createCard(item);
       cardList.addItem(cardElement);
     },
   },
   ".cards__list"
 );
-cardList.renderItems();
+
+api
+  .getInitialCards()
+  .then((cards) => {
+    if (Array.isArray(cards)) {
+      cardList.renderItems(cards);
```

Comment:
* The cards from the server are not displayed on the page when I start the project


Please, change `renderItems` inside `Section` so that it could accept the array


```js

  renderItems(items) {
    items.forEach((item) => {
      this._renderer(item);
    });
  }

```